src/components/transactions/Faucet/FaucetActions.tsx

```diff
@@ -13,13 +13,13 @@ export type FaucetActionsProps = {
 };
 
 export const FaucetActions = ({ poolReserve, isWrongNetwork, blocked }: FaucetActionsProps) => {
-  const { faucetService } = useTxBuilderContext();
+  const mint = useRootStore((state) => state.mint);
   const { currentAccount } = useWeb3Context();
```

Comment:
I noticed in a few places useWeb3Context is still used to get the currentAccount, should this be refactored to use the root store?

Reply:
This pr touches 91 files and there have been quite massive merge conflicts down the road, so at some point we decided to "touch as little as possible".

With the currentAccount as long as aave-interface relies on outdated web3-react there's not much advantage from one over the other. The currentAccount in zustand is just the synced currentAccount from context.
Might make sense to refactor down the road, but for now i'd keep it (same reason for still having appDataProvider).